articles/client-platforms/react/05-linking-accounts.md

```diff
@@ -0,0 +1,372 @@
+
+Now you're able to update the `LinkedAccountsList` component to render a __Link Account__ button using the `LinkAccountService.link` method:
+
+```javascript
+/* ===== ./src/components/LinkedAccount/LinkedAccountsList.js ===== */
+import React, { PropTypes as T } from 'react'
+import LinkAccountService from 'utils/LinkAccountService'
+...
+
+export class LinkedAccountsList extends React.Component {
+  ... //omitting some code
+
+  render(){
+    const { profile, auth } = this.props
+    const linker = new LinkAccountService(auth) // initializing the new helper
+    let items = []
+    if (profile) {
+      items = profile.identities.map(identity => {
+        return (<LinkedAccountItem {...this.props} identity={identity} />)
+      })
+    }
+
+    return (
+      <div className={styles.root}>
+        <h3>Linked Accounts</h3>
+        <ListGroup>{items}</ListGroup>
+        // the button to call auth0lock and show sign in window
+        <Button onClick={linker.link} bsStyle="primary">Link Account</Button>
+      </div>
+    )
+  }
+}
+
+export default LinkedAccountsList;
+```
+
+Running the application, you'are now able to click on __Link Account__ button in the user home page and link a _facebook_ or _google_ account to it. After the successful link, the new identity is showed in the __Linked Accounts__ list.
+
+## 3. Un-Linking Accounts
+
+You can also dissociate a linked account just hitting the [unlink user account endpoint](https://auth0.com/docs/api/management/v2#!/Users/delete_provider_by_user_id) in auth0 api. You need to send the primary user_id, and the provider/user_id of the identity you want to unlink. Let's update `AuthService` to, besides a `LinkAccount` method, provide also an `UnlinkAccount` method:
+
+```javascript
+/* ===== ./src/utils/AuthService.js ===== */
+import { EventEmitter } from 'events'
+import Auth0Lock from 'auth0-lock'
+
+export default class AuthService extends EventEmitter {
+  ... //omitting some code
+  unlinkAccount(identity){
+    // sends a delete request to unlink the account identity
+    this.fetchApi(`<%= "identities/${identity.provider}/${identity.user_id}" %>`, {
+      method: 'DELETE'
+    })
+    .then(identities => {
+      const profile = this.getProfile()
+      this.setProfile({...profile, identities}) // updates profile identities
+    })
+  }
+}
+```
+
+The method `unlinkAccount` receives the identity object, sending a DELETE request to the identities endpoint and, in case of success, updates the stored profile with the current identities list. As setProfile emmits the event `profile_updated`, the view components will be properly updated.
```

Comment:
@flaviogranero 
`emmits` should be `emits`. 
"DELETE" should be `DELETE`.
"setProfile" should be `setProfile`.